src/main/java/com/arpnetworking/commons/test/ThreadLocalBuildableTestHelper.java

```diff
@@ -120,5 +120,20 @@
         }
     }
 
+    private static <T> Optional<Field> getThreadLocalBuilderField(
```

Comment:
Do the other use cases for `getField` not want to process inherited fields too?

Reply:
Good point! Integrated this into `getField`.